data/README.md

```diff
@@ -34,4 +34,16 @@ jq '{
     members: .members.nodes | map({github: .login, name: (.name // "")})
   })
 }' data/github-teams.json > data/github-teams-simplified.json
-```
\ No newline at end of file
+```
+
+## Platform Packages
+
+To generate the list of platform packages, get the list of packages on the last readme of the platform
+(e.g., https://github.com/coq/platform/blob/2025.01.0/doc/README~8.20~2025.01.md) and place it 
+in the following files (by level): 
+gen_tools_base.html gen_tools_ide.html gen_tools_full.html gen_tools_extended.html gen_tools_optional.html gen_tools_dependency.html
+
+Then run the script from the `data` folder
+```bash
+../scripts/cat_tools.sh 
+```
```

Comment:
Given that YAML is a superset of JSON, and given that these README files themselves are auto-generated, I'm pretty sure that we can improve this workflow with the help of @rocq-prover/platform-maintainers by directly generating JSON files from the source that produces these README files.